src/content/loaders/index.md

```diff
@@ -43,12 +44,12 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 ## Templating
 
 - [`html-loader`](/loaders/html-loader) Exports HTML as string, require references to static resources
-- `pug-loader` Loads Pug templates and returns a function
-- `jade-loader` Loads Jade templates and returns a function
-- `markdown-loader` Compiles Markdown to HTML
+- [`pug-loader`](https://github.com/pugjs/pug-loader) Loads Pug templates and returns a function
+- [`jade-loader`](https://github.com/pugjs/pug-loader) Loads Jade templates and returns a function
```

Comment:
wrong url

Reply:
Aparently, they have one package for both and package `jade-loader` does not exist.

Reply:
```suggestion
```